### PR TITLE
Longer functional test support

### PR DESCRIPTION
**Pull request by vasukalariya**
# Description
Summarize the change.
- Multi-process ingestion should ingest unique data. Each process would have a unique seed.
- Added timestamp support for functional Test. Now, data should be ingested with a unique timestamp. 3 hour slot is given to each process.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?
- Ingested with same configuration multiple times and check that data is same.

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by nkunal — COMMENTED** [review]
looks good

File: tools/sigclient/cmd/sigclient.go [hunk 1]
```diff
@@ -126,10 +127,18 @@ var functionalTestCmd = &cobra.Command{
 		maxVariableCols := 20
 		sleepDuration := 15 * time.Second
 
+		if longer {
+			totalEvents = 50_000_000
+			batchSize = 100
+			numIndices = 20
+			processCount = 10
```
Comment (Macbeth98) [inline tools/sigclient/cmd/sigclient.go:134]: Are we not going set the process count and num indices through config?

File: tools/sigclient/functionalQueries/functionalQueries.yml [hunk 2]
```diff
@@ -6,10 +6,10 @@ grpby_1col_4stats.json
 search_1col_fields.json
 rare.json
 sort_tail_eval_newField.json
-tail_stats.json
+# tail_stats.json
 stats_groupby_with_eval.json
-fillnull_with_eval.json
-search_with_eval.json
+# fillnull_with_eval.json
+# search_with_eval.json
```
Comment (Macbeth98) [inline tools/sigclient/functionalQueries/functionalQueries.yml:12]: `search_with_eval.json` will work now with the my latest PR that got merged.

File: tools/sigclient/cmd/sigclient.go (hunk 1 above)
Comment (vasukalariya) [reply tools/sigclient/cmd/sigclient.go:134]: No, we do not want to expose these params.

**Review by nkunal — APPROVED** [review]

**Pull request merged**